### PR TITLE
[2861] Reduce avatar initials font size

### PR DESCRIPTION
**Pull request by bychkovdmitry**
https://github.com/GetStream/stream-chat-android/issues/2861

### 🎯 Goal

Reduce the font size of user initials where necessary.

### 🛠 Implementation details

Reduced the font size of fallback initials to:

| Component | Avatar fallback text size |
| --- | --- |
| MessageItem | 10sp |
| QuotedMessage | 10sp |
| ThreadParticipants | 7sp |

### 🎨 UI Changes

| Before | Avatar |
| --- | --- |
|photo_2021-12-31 17 47 57 | photo_2021-12-31 17 48 00|

#### General
- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] Assigned a person / code owner group (required)
- [x] Thread with the PR link started in a respective Slack channel (#android-team or #compose-chat-sdk-team) (required)
- [x] PR targets the `develop` branch

#### Code & documentation
- [x] Changelog is updated with client-facing changes
~~- [ ] New code is covered by unit tests~~
- [X] Comparison screenshots added for visual changes
~~- [ ] Affected documentation updated (KDocs, docusaurus, tutorial)~~

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### 🎉 GIF

_Please provide a suitable gif that describes your work on this pull request_

### CODE REVIEW

File: stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/ThreadParticipants.kt [hunk 1]
```diff
@@ -62,12 +62,14 @@ public fun ThreadParticipants(
...
-            ImageAvatar(
+            Avatar(
                 modifier = itemPadding
                     .zIndex(itemPosition)
                     .size(itemSize)
                     .border(border = borderStroke, shape = ChatTheme.shapes.avatar),
-                painter = painter
+                imageUrl = user.image,
+                initials = user.initials,
+                textStyle = ChatTheme.typography.captionBold.copy(fontSize = 7.sp),
```
Comment (bychkovdmitry) [inline stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/ThreadParticipants.kt:72]: I don't want to introduce a new font just for the one use case. Seems like an exceptional use case and in general a font shouldn't be that small.

**Review by skydoves — APPROVED** [review]
Looks good to me!

File: stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/ThreadParticipants.kt (hunk 1 above)
Comment (filbabic) [reply stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/ThreadParticipants.kt:72]: Agreed - this should be fine for now.

**Review by MarinTolic — APPROVED** [review]
### ☑️Reviewer Checklist
- [x] UI Components sample runs & works
- [x] Compose sample runs & works
- [x] UI Changes correct (before & after images)
- [x] Bugs validated (bugfixes)
- [x] New feature tested and works
- [x] Release notes and docs clearly describe changes
- [x] ~~All code we touched has new or updated KDocs~~ (note necessary)

**Review by filbabic — APPROVED** [review]
Checked everything and looks good!

### ☑️Reviewer Checklist
- [x] UI Components sample runs & works
- [x] Compose sample runs & works
- [x] UI Changes correct (before & after images)
- [x] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [x] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

**Pull request merged**